state/modeluser.go

```diff
@@ -264,10 +264,10 @@ func (st *State) modelQueryForUser(user names.UserTag, isSuperuser bool) (mongo.
 }
 
 type ModelAccessInfo struct {
-	Name           string `bson:"name"`
-	UUID           string `bson:"_id"`
-	Owner          string `bson:"owner"`
-	Type           ModelType
+	Name           string    `bson:"name"`
+	UUID           string    `bson:"_id"`
+	Owner          string    `bson:"owner"`
+	Type           ModelType `bson:"type"`
 	LastConnection time.Time
```

Comment:
Why no tag for LastConnection? Because we don't query for it?

Reply:
Yup